tasks/mpi/drozhdinov_d_sum_cols_matrix/include/ops_mpi.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Nesterov Alexander
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <boost/mpi/collectives.hpp>
+#include <boost/mpi/communicator.hpp>
+#include <memory>
+#include <numeric>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "core/task/include/task.hpp"
+
+namespace drozhdinov_d_sum_cols_matrix_mpi {
+
+std::vector<int> getRandomVector(int sz);
+int makeLinCoords(int x, int y, int xSize);
+std::vector<int> calcMatSumSeq(const std::vector<int>& matrix, int xSize, int ySize, int fromX, int toX);
+std::vector<int> calculateMatrixSumSequentially(const std::vector<int>& matrix, int xSize, int ySize);
+class TestMPITaskSequential : public ppc::core::Task {
+ public:
+  explicit TestMPITaskSequential(std::shared_ptr<ppc::core::TaskData> taskData_) : Task(std::move(taskData_)) {}
+  bool pre_processing() override;
+  bool validation() override;
+  bool run() override;
+  bool post_processing() override;
+
+ private:
+  std::vector<int> input_;
+  std::vector<int> res;
+  int cols{};
+  int rows{};
+};
+
+class TestMPITaskParallel : public ppc::core::Task {
+ public:
+  explicit TestMPITaskParallel(std::shared_ptr<ppc::core::TaskData> taskData_) : Task(std::move(taskData_)) {}
+  bool pre_processing() override;
+  bool validation() override;
+  bool run() override;
+  bool post_processing() override;
+
+ private:
+  std::vector<int> input_, local_input_;
```

Comment:
```suggestion
  std::vector<int> input_;
``` Please remove unused variables

Reply:
Please remove unused variables

Reply:
done